generators/client/templates/vue/package.json.ejs

```diff
@@ -110,7 +110,7 @@ limitations under the License.
     "postcss-import": "12.0.1",
     "postcss-loader": "3.0.0",
     "postcss-url": "8.0.0",
-    "prettier": "2.0.5",
+    "prettier": "2.1.1",
```

Comment:
We should use the constant like angular.
```suggestion
    "prettier": "<%= PRETTIER_VERSION %>",
```

Reply:
And you should run `npm run lint-fix`

Reply:
I did that but there are not modifications 🧐

Reply:
npm ci?

Reply:
no updates 😕

Reply:
Change prettier:check script to use —debug-check.
This should print the error at actions.

Reply:
![image](https://user-images.githubusercontent.com/9989211/91438113-277efd80-e86b-11ea-82cf-a53bf6551702.png)

there is a warn on test/needle-api/needle-server-liquibase.spec.js but I don't see the problem

Reply:
I updated a file (package.json.ejs) and run: npm run lint-fix
=> test/needle-api/needle-server-liquibase.spec.js was updated :)